pkg/env/vulnerability_management.go

```diff
@@ -9,4 +9,7 @@ var (
 
 	// ReportCustomEmailBodyMaxLen sets the maximum allowed length for custom email body in vulnerability report emails
 	ReportCustomEmailBodyMaxLen = RegisterIntegerSetting("ROX_REPORT_CUSTOM_EMAIL_BODY_MAX_LEN", 1500)
+
+	// ImageCVEEdgeJoinWorkaround is to enable/disable the temporary workaround to fix image_cve_edge search issue (ROX-17252)
+	ImageCVEEdgeJoinWorkaround = RegisterBooleanSetting("ROX_IMAGE_CVE_EDGE_JOIN_WORKAROUND", false)
```

Comment:
Do we want to default to on?  Probably shouldn't call it `_WORKAROUND`.

Reply:
Yeah I was initially planning to turn it on after the bug bash. But I will do it here. Renaming it to ROX_IMAGE_CVE_EDGE_CUSTOM_JOIN .